acm/observables/emc/density_split_correlation.py

```diff
@@ -2,32 +2,34 @@
 import numpy as np
 from pathlib import Path
 from .base import BaseObservableEMC
+import matplotlib.pyplot as plt
 from acm.utils.default import cosmo_list # List of cosmologies in AbacusSummit
 from acm.utils.xarray_data import dataset_to_dict
+from acm.utils.plotting import set_plot_style
 
-class DensitySplitCorrelationFunctionMultipoles(BaseObservableEMC):
+class DensitySplitGalaxyCorrelationFunctionMultipoles(BaseObservableEMC):
     """
     Class for the Emulator's Mock Challenge density-split correlation
     function multipoles.
     """
     def __init__(self, **kwargs):
-        super().__init__(stat_name='dsc_conf', **kwargs)
-        self.n_test = 6*100 #Override default number of test samples 
+        super().__init__(stat_name='ds_xiqg', n_test=6*3, **kwargs)
```

Comment:
Suggestion: `n_test` could be passed as an argument of __init__ to allow overriding if needed ?

```suggestion
    def __init__(self, n_test=6*3, **kwargs):
        super().__init__(stat_name='ds_xiqg', n_test=n_test, **kwargs)
```

> (Also valid for other EMC Observables)